packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -426,12 +421,12 @@ export default compose(
 					);
 				},
 				duplicateBlock() {
-					return duplicateBlocks( clientIds );
+					return duplicateBlocks( [ clientId ] );
 				},
 				onMoveDown: ( ...args ) =>
-					moveBlocksDown( clientIds, rootClientId, ...args ),
+					moveBlocksDown( [ clientId ], rootClientId, ...args ),
 				onMoveUp: ( ...args ) =>
-					moveBlocksUp( clientIds, rootClientId, ...args ),
+					moveBlocksUp( [ clientId ], rootClientId, ...args ),
```

Comment:
A side note (nothing we need to address in this PR): seems like `createOnMove()` no longer needs to cast the client IDs to an array, since it already expects only an array.